libraries/AP_Scripting/examples/proximity_test.lua

```diff
@@ -0,0 +1,27 @@
+-- This script checks Proximity
+
+function update()
+  sensor_count = proximity:num_sensors()
+  gcs:send_text(0, string.format("%d proximity sensors found.", sensor_count))
+
+  if sensor_count > 0 then
+    object_count = proximity:get_object_count()
+    gcs:send_text(0, string.format("%d objects found.", object_count))
+
+    closest_angle, closest_distance = proximity:get_closest_object()
+    if closest_angle ~= nil and closest_distance ~= nil then
```

Comment:
```suggestion
    if closest_angle and closest_distance then
```

You don't need to explicitly compare to 0, you can actually just use the values directly as nil will look like a logical false, and the numbers will be true for the condition.